### ISSUE TITLE
UX: Create new template (not based on exisiting)

### ISSUE THREAD

**Issue by oldhoyte** [cherrytwist/Coordination#746]
**Is your feature request related to a problem? Please describe.**
As a hub admin, I want to create a canvas template directly from the settings, without creating the Canvas first and then selecting it as a template. 

**Describe the solution you'd like**
Two options for the Canvas templates settings: 
1. select existing Canvas as template (hub, challenge and opportunity canvases).
2. create template directly

![image.png](https://images.zenhubusercontent.com/6200e102047aab2b04cbfbce/eea3e560-94b5-4506-99ab-5451376c2000)

**Comment by techsmyth** [comment]
This would then result in a canvas template **without** a a canvas? Or do you expect to create the canvas also directly on the canvas template?

**Comment by oldhoyte** [comment]
Yes, this would lead to the possibility to create a canvas template without needing any canvas in your hub yet, and without creating a canvas by making the template.
